content/general/concepts/mfa/mfa.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'MFA'
+Description: 'Multi-factor Authentication (MFA) is the use of more than one factor to prove the identity of a user.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Best Practices'
+  - 'Cybersecurity'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
```

Comment:
```suggestion
  - 'introduction-to-cybersecurity'
  - 'paths/full-stack-engineer-career-path'
```

Both of the these pages contain lessons and modules pertaining to user authentication/authorization.